src/main/java/eu/europeana/metis/sandbox/scheduler/XsltUrlUpdateScheduler.java

```diff
@@ -18,8 +18,7 @@ public XsltUrlUpdateScheduler(XsltUrlUpdateService xsltUrlUpdateService) {
     this.xsltUrlUpdateService = xsltUrlUpdateService;
   }
 
-  // "0 0 * * * *" = every hour of every day.
-  @Scheduled(cron = "0 0 * * * *")
+  @Scheduled(cron = "${sandbox.transformation.xslt-update-frequency}")
   public void updateDefaultXsltUrl() {
```

Comment:
remember to include a comment about this on the ticket, that there is a new property when the release is ready to not forget to include in jenkins.

Reply:
I mean in the yml properties, also you can define a default value if we forget.
```${sandbox.transformation.xslt-update-frequency:0 0 * * * *}```